java/samples/sample-code/src/main/java/com/microsoft/semantickernel/samples/documentationexamples/Prompts.java

```diff
@@ -13,30 +14,31 @@ public class Prompts {
 
     // Only required if AZURE_CLIENT_KEY is set
     private static final String CLIENT_ENDPOINT = System.getenv("CLIENT_ENDPOINT");
-    private static final String MODEL_ID = System.getenv().getOrDefault("MODEL_ID", "gpt-35-turbo-2");
+    private static final String MODEL_ID = System.getenv().getOrDefault("MODEL_ID",
+        "gpt-35-turbo-2");
```

Comment:
If we're going to go with Azure OpenAI model ids as the default, then we need to be consistent with all the examples. And we need to state in the readme that these are Azure OpenAI model ids and if you use OpenAI model ids, then you need to set MODEL_ID. I can take this on in a separate PR if you'd like. 

There is also the issue of different models. One example uses both chat completion and text embedding. Some of the examples are chat others text. Maybe we should have CHAT_MODEL_ID, EMBEDDING_MODEL_ID, IMAGE_MODEL_ID, etc.